### ISSUE TITLE
Add Categories to Product Page

### ISSUE THREAD

**Issue by niklashmbgr** [LudiFlow/capstone-project#13]
# Value Proposition

**As a** User 
**I want to** filter the products by different categories
**So that** I can find the product I am looking for faster. 

# Description

![Image](https://user-images.githubusercontent.com/107849116/185354561-806e39cb-54aa-4512-bd35-22b5be8aa8e6.png)

# Acceptance Criteria

- [ ] the user can filter by two different categories: tea or coffee
- [ ] by clicking on a filter button only products matching the selected category are shown
- [ ] when the user opens the app the "show all"-category is selected by default and all products are shown 

# Tasks

- [ ] create new feature `branch` ("addCategories")
- [ ] add three different buttons to the Product Page (coffee/tea/all)
- [ ] add filter.function to 'coffee + tea'-button
- [ ] add filter.function(reset) to 'all'-button 

# Complexity

M

**Comment by ahohnsen** [comment]
Hey,

I would rephrase the value proposition a bit. The feature you wanna implement is a filter for the user:

As a user
I want to filter the products by different categories
so that I can find the product I am looking for faster.

Please try to phrase your acceptance criteria without "should", "could" and as specific as possible, e.g.:

- The user can filter by two different categories: tea or coffee
- By clicking on a filter button only products matching the selected category are shown
- By clicking on the "show all" button all products are displayed
- When the user opens the app no filter is selected and all products are shown

Have a thought about the last point 🙂 you could also have by default the category "show all" selected.

**Comment by A11yship** [comment]
Looks good now
